src/main/java/com/revature/model/Interview.java

```diff
@@ -25,41 +24,35 @@
  */
 @Entity
 @Table(name = "interviews")
+
```

Comment:
Why is there a blank line here? Try not to have a change somewhere if it is not needed.

Reply:
And between methods as well.

Reply:
Below you removed blank lines that were already there. Between member variables/methods, a blank line serves to add readability, so those should remain. The above line is different, as generally there are no blank lines between an annotation and the thing being annotated.

Reply:
Changes are made as per suggestion